src/config.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  dataUrl: 'https://spreadsheets.google.com/feeds/list/1CyPozeKhmOuIaVnKDQAUKKsvA9R4F20hEQ3MSGWczP8/1/public/full?alt=json',
```

Comment:
I figured putting these in one spot, it shouldn't be too hard to externalize stuff with https://www.snowpack.dev/#environment-variables

Reply:
@omawhite is working on that :)

Reply:
Based on this it looks like we may need to use different names than what I proposed to @experimatt

Reply:
You should be able to use the same names, we'll just change it from that hard coded string to something different. (Not sure where TRANSLATION_URL goes yet)

```
const { MAPBOXGL_ACCESS_TOKEN, DATA_URL, TRANSLATION_URL } = import.meta.env;

export default {
  dataUrl: DATA_URL,
  accessToken: MAPBOXGL_ACCESS_TOKEN,
  translationUrl: TRANSLATION_URL
}
```

Reply:
Oh I see. Maybe if I'd read the instructions I'd realize that you do have to change them. Apologies 🙇